api/tests/functional-tests/backend/metrics/test_classification.py

```diff
@@ -365,6 +367,7 @@ def test_compute_confusion_matrix_at_grouper_key_and_filter(
             groundtruth="bird", prediction="dog", count=1
         ),
     ]
+    assert cm
```

Comment:
all of these assertion statements are used to avoid typing issues where the return type is optional